daemon/daemon.go

```diff
@@ -951,6 +959,13 @@ func (daemon *Daemon) Shutdown() error {
 // Mount sets container.BaseFS
 // (is it not set coming in? why is it unset?)
 func (daemon *Daemon) Mount(container *container.Container) error {
+	daemon.mountingContainers++
```

Comment:
This can't be done as it's racey to modify the counter in different goroutines.
Even if we used an atomic counter I'm not sure it would resolve the problem.